packages/test/test-end-to-end-tests/src/test/error.spec.ts

```diff
@@ -99,12 +100,15 @@ describeNoCompat("Errors Types", (getTestObjectProvider) => {
         }
     });
 
+    //* test CreatecontainerError around errorType precedence, and whether the error is LoggingError or not
+    //* test createOdspNetworkError around the props that should be added
+
     it("GeneralError Logging Test", async () => {
         const err = {
             userData: "My name is Mark",
             message: "Some message",
         };
-        const iError = (CreateContainerError(err) as any) as LoggingError;
+        const iError = (CreateContainerError(err) as any) as ILoggingError;
```

Comment:
you've bundled a lot of refactoring in this PR, and we split that out into a different one